cluster-autoscaler/main.go

```diff
@@ -314,6 +314,9 @@ func run(healthCheck *metrics.HealthCheck) {
 	// Start updating health check endpoint.
 	healthCheck.StartMonitoring()
 
+	// Start components running in background.
+	autoscaler.Start()
```

Comment:
Either handle error here or drop it from signature

Reply:
done